### PR TITLE
Provide Free tickets

### PR DESCRIPTION
**Pull request by skamoen**
This PR adds the requests to checkout orders, without going to the payment provider. As part of this functionality, it's possible to add Orders for users as Admin, and a FREE ticketType was added.

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/controller/OrderRestController.java [hunk 1]
```diff
@@ -128,14 +147,21 @@ public Order getOrderById(@PathVariable Long orderId) {
...
-        //TODO: Implement Paymentprovider calls here.
         String paymentUrl = orderService.requestPayment(orderId);
         HttpHeaders headers = new HttpHeaders();
         headers.setLocation(new URI(paymentUrl));
 
         return createResponseEntity(HttpStatus.OK, headers, "Please go to " + paymentUrl + " to finish your payment");
     }
 
+    @PreAuthorize("hasRole('ADMIN')")
+    @RequestMapping(value = "/orders/{orderId}/checkout", method = RequestMethod.POST, params = "admin")
```
Comment (martijn9612) [inline src/main/java/ch/wisv/areafiftylan/controller/OrderRestController.java]: Why are we using this as url, the admin parameter is a bit out of place. Why not change the url to it's purpose, `/orders/{orderId}/approve`? A javadoc comment would also be very helpful in this case.

File: src/main/java/ch/wisv/areafiftylan/service/OrderService.java [hunk 2]
```diff
@@ -58,15 +61,19 @@
 
     Order updateOrderStatus(Long orderId);
 
+    void adminApproveOrder(Long orderId);
```
Comment (martijn9612) [inline src/main/java/ch/wisv/areafiftylan/service/OrderService.java]: Please add a javadoc comment.

**Comment by martijn9612** [comment]
Changes look very good, nice work!

File: src/main/java/ch/wisv/areafiftylan/controller/OrderRestController.java [hunk 3]
```diff
@@ -61,6 +60,12 @@ public OrderRestController(OrderService orderService) {
         HttpHeaders headers = new HttpHeaders();
         User user = (User) auth.getPrincipal();
 
+        // You can't buy non-buyable Tickts for yourself, this should be done bia the createAdminOrder() method.
```
Comment (TimvdLippe) [inline src/main/java/ch/wisv/areafiftylan/controller/OrderRestController.java]: Bia -> via

**Comment by martijn9612** [comment]
I believe the `/orders` part of the route can also be extracted in `orderRestController`. By bad for seeing this not earlier.

**Comment by skamoen** [comment]
This route messes that up:
`@RequestMapping(value = "/users/{userId}/orders", method = RequestMethod.POST)`

**Comment by martijn9612** [comment]
Looks good!

**Pull request merged**